src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,27 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Todo } from '../../types/Todo';
+import { Status } from '../../enums/Status';
+
+export const TodoList = () => {
+  const { todos, filter } = useContext(TodosContext);
+
+  let filteredTodos = todos;
+
+  if (filter === Status.Active) {
+    filteredTodos = todos.filter(todo => todo.completed !== Status.Completed);
+  }
+
+  if (filter === Status.Completed) {
+    filteredTodos = todos.filter(todo => todo.completed === Status.Completed);
+  }
+
+  return (
+    <ul className="todo-list" data-cy="todoList">
+      {filteredTodos && filteredTodos.map(
```

Comment:
Here, `filteredTodos &&` is not necessary because `filteredTodos` is never `undefined` or `null`. It's always an array, even if it's empty, so you can safely map over it.